app/models/drug.rb

```diff
@@ -0,0 +1,37 @@
+class Drug < ActiveYaml::Base
+  set_root_path "config/data"
+  set_filename "drugs"
+
+  CREATED_TIME ||= File.ctime("config/data/drugs.yml").in_time_zone("UTC")
+  UPDATED_TIME ||= File.mtime("config/data/drugs.yml").in_time_zone("UTC")
+
+  CATEGORIES = {
+    hypertension_ccb: "Hypertension: CCB",
+    hypertension_arb: "Hypertension: ARB",
+    hypertension_diuretic: "Hypertension: Diuretic",
+    hypertension_ace: "Hypertension: ACE Inhibitor",
+    hypertension_other: "Hypertension: Other",
+    diabetes: "Diabetes",
+    other: "Other"
+  }.freeze
+
+  FREQUENCIES = {
+    once_a_day: "Once a day",
+    twice_a_day: "Twice a day",
+    thrice_a_day: "Thrice a day"
+  }.freeze
```

Comment:
Similar to the `CATEGORIES` map above, a version of this is in the PrescriptionDrug class, should get folded into this class instead. Also using these representations instead of "OD", "BD" etc, since those strings are not applicable everywhere (eg: Ethiopia), and need to be i18n-ized. Picked this clearer representation since we'll i18n-ize anyway.

Reply:
Can this just be a list, since the values in the hash don't represent anything important? I think `PrescriptionDrug` has a hash because it's an `enum` and we also use the values to display.

Reply:
We could, I put in a hash here so we can turn it into an `enum` when this gets promoted to an AR backed model